src/MSRIOGroup.cpp

```diff
@@ -108,11 +108,21 @@ namespace geopm
             parse_json_msrs(data);
         }
 
+        m_hwp_is_enabled = hwp();
         register_signal_alias("TIMESTAMP_COUNTER", "MSR::TIME_STAMP_COUNTER:TIMESTAMP_COUNT");
+
+        if (m_hwp_is_enabled)
+        {
+            // TODO: make a combined signal of min, max, and desired.  This is handled in write_control
+            register_signal_alias("CPU_FREQUENCY_CONTROL", "MSR::HWP_REQUEST:MINIMUM_PERFORMANCE");
```

Comment:
Currently mapping CPU_FREQUENCY_CONTROL to MINIMUM_PERFORMANCE only.  A better long term is a combinedsignal of the MINIMUM and MAXIMUM_PERFORMANCE fields.